features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt.h

```diff
@@ -0,0 +1,282 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015-2016 Nuvoton
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+#ifndef MBEDTLS_DES_ALT_H
+#define MBEDTLS_DES_ALT_H
+
+#if !defined(MBEDTLS_CONFIG_FILE)
```

Comment:
@RonEld Please check fix in https://github.com/ARMmbed/mbed-os/pull/4925/commits/eac27077834e1cad4de6842a8f65839f46472da3.